src/Homely.AspNetCore.Mvc.Helpers/DateTimeConverter.cs

```diff
@@ -0,0 +1,35 @@
+using System;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+
+namespace Homely.AspNetCore.Mvc.Helpers
+{
+    // based on REF: https://stackoverflow.com/a/58103218/30674
+    // Also info on default behavior for DateTimes with System.Text.Json: https://docs.microsoft.com/en-us/dotnet/standard/datetime/system-text-json-support
+    public class DateTimeConverter : JsonConverter<DateTime>
+    {
+        private readonly string _dateTimeFormat;
+
+        public DateTimeConverter(string dateTimeFormat)
+        {
+            if (string.IsNullOrWhiteSpace(dateTimeFormat))
+            {
+                throw new ArgumentException($"'{nameof(dateTimeFormat)}' cannot be null or whitespace.", nameof(dateTimeFormat));

```

Comment:
any reason `throw new ArgumentNullException(nameof(dateTimeFormat))` isn't enough?

Reply:
yep! great question -> i clicked `<ctrl> + <.>` in VS and it auto completed/filled this out. I didn't even notice the exception was an `AE` (instead of an `ANE`).

I did notice that there was some pre-loaded TEXT in the message, though .. which is why i missed the AE/ANE. i'll update.

Reply:
🔧